### PR TITLE
Upgrade APIScan version

### PR DESCRIPTION
**Pull request by adityapatwardhan**
# PR Summary

APIScan tool version must be upgraded.

## PR Context

Using older version is not compliant.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Review by TravisEz13 — APPROVED** [review]

File: tools/releaseBuild/azureDevOps/templates/compliance.yml [hunk 1]
```diff
@@ -99,7 +99,7 @@ jobs:
```
Comment (PoshChan) [inline tools/releaseBuild/azureDevOps/templates/compliance.yml:99]: @adityapatwardhan, your last commit had 5 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/testing-guidelines/TestRoadmap.md.https://github.com/PowerShell/PowerShell/issues?utf8=%E2%9C%93&q=is%3Aissue%20%23testability%20 should work
```powershell
The response ended prematurely.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -Uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CODE_OF_CONDUCT.md.https://opensource.microsoft.com/codeofconduct/ should work
```powershell
Connection timed out
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -Uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CODE_OF_CONDUCT.md.https://opensource.microsoft.com/codeofconduct/faq/ should work
```powershell
Connection timed out
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -Uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://opensource.microsoft.com/codeofconduct/ should work
```powershell
Connection timed out
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -Uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://opensource.microsoft.com/codeofconduct/faq/ should work
```powershell
Connection timed out
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -Uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```

**Comment by adityapatwardhan** [comment]
@PoshChan please rebuild all

**Comment by PoshChan** [comment]
@adityapatwardhan, did not find matching build context: `PowerShell-CI-Windows`; allowed contexts: PowerShell-CI-static-analysis

File: tools/releaseBuild/azureDevOps/templates/compliance.yml (hunk 1 above)
Comment (PoshChan) [inline tools/releaseBuild/azureDevOps/templates/compliance.yml:99]: @adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32 should work
```powershell
Failed to complete request to "https://powershell.visualstudio.com/PowerShell/_build/latest?definitionId=32". Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```

**Comment by adityapatwardhan** [comment]
@PoshChan please retry static

**Comment by PoshChan** [comment]
@adityapatwardhan, successfully started retry of `PowerShell-CI-static-analysis`

**Pull request merged**